terraform/module-lemp/redis.tf

```diff
@@ -41,14 +41,15 @@ resource "aws_elasticache_cluster" "redis" {
 }
 
 resource "aws_elasticache_subnet_group" "cache-subnet" {
-  name        = "cache-${var.project}-${var.vpc_id}-${var.env}"
+  name        = replace("cache-${var.project}-${var.vpc_id}-${var.env}", var.nameregex, "")
   count       = var.cache_subnet_group == "" && var.create_elasticache ? 1 : 0
   description = "redis cache subnet for ${var.project}-${var.env} ${var.vpc_id}"
   subnet_ids  = var.private_subnets_ids
 }
 
 output "elasticache_address" {
-  value = aws_elasticache_cluster.redis[0].cache_nodes[0].address
+  value = length(aws_elasticache_cluster.redis) > 0 ? aws_elasticache_cluster.redis[0].cache_nodes[0].address : ""
+  #value = aws_elasticache_cluster.redis[0].cache_nodes[0].address
```

Comment:
Is it meant to stay commented?
(Same question goes for the other files)

Reply:
I don't really know. It is still a workaround and we don't know if terraform will revert it or introduce a "new feature".